src/assets/App.css

```diff
@@ -1,11 +1,163 @@
 .App {
-  text-align: center;
+  padding: 4%;
+  background-color: #fafafa;
 }
 
 .book {
   display: grid;
   align-items: center;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
-  margin: 2em;
-  border: solid 1px black;
+  grid-template-columns: 2fr 1fr 1fr;
+  background-color: #fff;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+  padding: 2rem;
+  margin-bottom: 0.938rem;
+}
+
+.hidden {
+  display: none;
+}
+
+.book-info {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  text-align: left;
+}
+
+.book-info h5 {
+  font: 700 0.875rem "Montserrat", sans-serif;
+  color: #121212;
```

Comment:
Please use CSS variables in order to avoid repetition of this kind of rules that you use, this applies to any other repetition like fonts, other colors. Just check the ones that are repeated a lot and make variables as necessary to refactor the code and make it reusable. Other than that, your milestone is perfect.

- [ ] In the stylesheets of the app, make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
  - no repeating of the same rule